sigmadsp/application/backend.py

```diff
@@ -151,6 +159,9 @@ def worker(self):
             if isinstance(request, WriteRequest):
                 self.dsp.write(request.address, request.data)
 
+            if isinstance(request, SafeloadRequest):
+                self.dsp.safeload(request.address, request.data)
+
             elif isinstance(request, ReadRequest):
                 payload = self.dsp.read(request.address, request.length)
 
```

Comment:
We may add an `else` case that raises an error, e.g. `raise TypeError(f"Unknown command type {type(request)}.")`.